install/install.php

```diff
@@ -100,6 +100,31 @@ public function install(Migration $migration, $args = []): bool {
       $task = new CronTask();
       PluginFormcreatorIssue::cronSyncIssues($task);
 
+      // Add rights
```

Comment:
The whole logic after this line to set rights in profiles should be in a dedicated method.

Also, rights must be added when upgrading the plugin. I don't think it is possible to factorize your code to add rights below; you will have to copy it in upgrade_to_2_14.php